debian/pop-default-settings.postinst

```diff
@@ -83,6 +83,15 @@ case "$1" in
             fi
         done
 
+        shopt -s nullglob
+        for file in /etc/apt/sources.list /etc/apt/sources.list.d/*.list
+        do
+            # Convert to apt.pop-os.org mirror, if using us.archive.ubuntu.com
+            sed -i 's|us.archive.ubuntu.com/ubuntu|apt.pop-os.org/ubuntu|' "${file}"
```

Comment:
So, this won't change anything if a user has switched to another Ubuntu mirror?

If we intend for our mirror to be anything other than an exact up-to-date copy of Ubuntu's archive, this seems like it could be a problem.

Reply:
That's correct. And we don't intend it to be any different - it is a legit Ubuntu mirror. The user is still free to change it after this.

This only changes the people with the default mirror.